optuna/multi_objective/samplers/_motpe.py

```diff
@@ -183,7 +183,7 @@ def _create_trial(mo_trial: "multi_objective.trial.FrozenMultiObjectiveTrial") -
         warnings.simplefilter("ignore", ExperimentalWarning)
         trial = create_trial(
             state=mo_trial.state,
-            values=mo_trial.values,
+            values=mo_trial.values,  # type: ignore
```

Comment:
```python
tuple(trial.intermediate_values.get(i) for i in range(n_objectives))
```

`FrozenMultiObjectiveTrial.values` is not `Optional[Sequence[float]]`. I simply ignore this mismatch because `optuna.multi_objective` is entirely deprecated.

https://github.com/optuna/optuna/blob/c4791fc0a67d1eb03ef650e583bf05f06af41dc4/optuna/multi_objective/trial.py#L279